packages/databricks-vscode/src/configuration/models/ConfigModel.ts

```diff
@@ -131,6 +131,9 @@ export class ConfigModel implements Disposable {
             }),
             this.bundlePreValidateModel.onDidChange(async () => {
                 await this.readTarget();
+                // We fire the target change emitter because auth info might have changed,
+                // which we handle exactly the same way as a target change.
+                this.onDidChangeTargetEmitter.fire();
```

Comment:
Seems a bit hacky - we will either emit an event without target actually changing, or emit the event twice. Both cases can lead to some bugs down the road, when we add more listeners for this event. 

Can we just listen to `configModel.jonDidChangeKey("host")` on the connection manager side? And eventually listen to more keys, like "authParams", when we support them. We even have a TODO over there about it

Reply:
Yes. That actually works better. Will update.